Effisense/sensor-dashboard
# Scan QR, validate QR code against SPAN API, and redirect to `/sensors/create`

Closes: #25 

## Checklist


## Changelog

- Add tests for `SpanApiPayloadSchema`
- Validate QR code
- If success, redirect to `/sensors/create`
- If error, display toast with error

## How to test

- Navigate to `/scan`
- Use your phone to emulate QR code, and the desktop camera to scan
- Scan a wrong QR code. You can use [this one](https://www.investopedia.com/thmb/hJrIBjjMBGfx0oa_bHAgZ9AWyn0=/1500x0/filters:no_upscale():max_bytes(150000):strip_icc()/qr-code-bc94057f452f4806af70fd34540f72ad.png)
- Observe that you get an error message as a toast
- Scan a correct QR code (see Slack). 
- Observe that you get a success toast and get redirect to `/sensors/create`
